import-service/handlers/import-products-file.js

```diff
@@ -0,0 +1,18 @@
+import { handleRequestResponseDecorator } from '../../lib/response';
+import { BadRequestError } from '../../lib/error';
+
+import ImportService from '../services/import';
+
+export const getImportProductSignedUrl = handleRequestResponseDecorator(
+  async function(event) {
+    const { queryStringParameters: { name } = {} } = event;
```

Comment:
If I do not pass any query parameters, then queryStringParameters=null by spec and this line will fail.

Just try https://vapcj1zvui.execute-api.eu-west-1.amazonaws.com/dev/import and you'll get 500:
{"error":{"message":"Something went wrong, please, try again later"}}

Try non-empty query (but not name): https://vapcj1zvui.execute-api.eu-west-1.amazonaws.com/dev/import?unknown=2 and you'll get correct 400